src/libasr/asr_verify.cpp

```diff
@@ -441,6 +441,32 @@ class VerifyVisitor : public BaseWalkVisitor<VerifyVisitor>
         verify_unique_dependencies(x.m_dependencies, x.n_dependencies,
                                    x.m_name, x.base.base.loc);
 
+        // Get the x parent symtab.
+        SymbolTable *sym = x.m_symtab->parent;
```

Comment:
Minor: I would name this as `parent_symtab`/`par_symtab`/`p_symtab`/`p_sym` so it is clear that it is the parent scope and not the current scope.
```suggestion
        SymbolTable *p_symtab = x.m_symtab->parent;
```